keyboards/keyhive/maypad/keymaps/keymap.c

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2019 codybender
```

Comment:
This file needs to be in a subfolder of `keyhive/maypad/keymaps/` in order to work, and the layout macro `LAYOUT_ortho_5x4` doesn't currently exist in your codebase, so this file wouldn't compile in its current state.

Reply:
I believe I have fixed this in my first commits today

Reply:
Ah my apologies, I see what you meant now. I forgot I left those files hanging in /keymaps/. I have resolved this now